magenta-lib/src/main/scala/magenta/deployment_type/ElasticSearch.scala

```diff
@@ -37,17 +37,16 @@ object ElasticSearch extends DeploymentType {
   ) { (pkg, resources, target) =>
```

Comment:
I considered removing this deployment type rather than including it in the refactor (https://github.com/guardian/riff-raff/pull/628), but it's unclear whether we can do this or not yet, so I've just ported the changes across for now.